osquery/tables/system/linux/deb_packages.cpp

```diff
@@ -26,6 +19,13 @@ extern "C" {
 #include <osquery/worker/ipc/platform_table_container_ipc.h>
 #include <osquery/worker/logging/glog/glog_logger.h>
 
+extern "C" {
```

Comment:
As soon as I added the BPF libraries and switched to C++17, these C headers started preventing the build from passing due to several definitions causing errors in the other C++ headers

Several things changed since then; I'll see if it can be removed

Reply:
I don't think it's possible to remove this and still be able to compile things; there are things inside those headers that will just break the build such as this macro

```
../libraries/cmake/source/libdpkg/src/lib/dpkg/macros.h:127:9: note: macro 'clamp' defined here
#define clamp(v, l, h) ((v) > (h) ? (h) : ((v) < (l) ? (l) : (v)))
```

colliding with [std::clamp](https://en.cppreference.com/w/cpp/algorithm/clamp)